microovn/metadata.yaml

```diff
@@ -18,6 +18,8 @@ requires:
     limit: 1
   certificates:
     interface: tls-certificates
+  ovsdb-cms:
+    interface: ovsdb-cms
```

Comment:
We might want to use this relation endpoint name for ourselves with a different interface implementation in the future.

Some history:
The original intent with the split between `ovsdb` and `ovsdb-cms` endpoint names was to allow the latter to bypass OVN Southbound DB RBAC rules, which is often needed for CMS systems connecting directly to the SB DB.

While we currently do not support Southbound DB RBAC, we might want to later.

Suggestion:
Our use of this relation is explicitly to allow use of external to MicroOVN SB DB, which is a different purpose than the original relation implementation.

So to make this intent more explicit and avoid future compatibility issues, should we name this something like `ovsdb-external` instead?